### DISCUSSION TITLE
API not working for message data

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#1937] [Help with using Postal]
Hello @adamcooke 

I am trying to use http API 
i have defined properly : X-Server-API-Key 

but getting the following error

<img width="563" alt="image" src="https://user-images.githubusercontent.com/32220165/161199089-11e72f85-4454-4e62-b045-95349e91e69b.png">

i have tried all GET and POST methods but no results

{
    "status": "parameter-error",
    "time": 0.0,
    "flags": {},
    "data": {
        "message": "`id` parameter is required but is missing"
    }
}

Please do advise if anything missing in it

**Comment by willpower232** [comment]
As per the documentation https://docs.postalserver.io/developer/api#general-api-instructions

You should be sending a JSON body to the API.
